jetty-io/src/main/java/org/eclipse/jetty/io/ManagedSelector.java

```diff
@@ -45,6 +46,7 @@
 import org.eclipse.jetty.util.component.DumpableCollection;
 import org.eclipse.jetty.util.log.Log;
 import org.eclipse.jetty.util.log.Logger;
+import org.eclipse.jetty.util.log.StdErrLog;
```

Comment:
Uh, no. We don't want a core class like ManagedSelector depend on a concrete implementation of our logging.
`StdErrLog` is never used directly apart in tests or configuration files, and must remain so.

Reply:
Why? It's part of util. I think it is reasonable to use common date formatting code, specially as it means date cache can be used. I guess I can move the method to a common logging class

Reply:
Not good because in light of JDK 9 modules `StdErrLog` will be a private class that we don't export, or something like that. Really, this is just formatting a date in case of dumping, the JDK has plenty of methods that do that, we don't need another utility method used only here.